### ISSUE TITLE
Create a pod with my Network Volume

### ISSUE THREAD

**Issue by lucianot54** [runpod/runpod-python#80]
**Is your feature request related to a problem? Please describe.**
My feature request is prompted by an issue. Currently, while deploying a pod through the web interface, I have the option to select the storage. However, using the CLI or Python library, I do not have the ability to choose the storage

**Describe the solution you'd like**
I would like to have the capability to create pods using the storage Network Volume. This feature would enable me to conveniently attach and detach network-attached storage to my pods, simplifying the management and scalability of my applications that rely on persistent data storage

**Describe alternatives you've considered**
I can do it only on the web interface, nothing else.

**Additional context**
I would like to create my pod using a script.

**Comment by RomainEconomics** [comment]
I agree, that would be a great feature. At the moment, the lack of this feature prevents me from using the lib for creating a pod

**Comment by justinmerrell** [comment]
I am working on merging a branch that will have this missing functionality.
